bitrise.yml

```diff
@@ -123,4 +122,7 @@ trigger_map:
 - pull_request_target_branch: "*"
   workflow: test
 - tag: "*"
-  workflow: beta
+  workflow: release
```

Comment:
Nemal by tu byt spis `enterprise`?

Reply:
V tom diffu to vypadá zmateně, ale je to v pořádku. Chci to postupně sjednotit na tři lane, workflow:

- `test`
- `enteprise`
- `release`